examples_tests/35.TAA/taa.frag

```diff
@@ -61,8 +90,8 @@ void main()
 	vec2 q_uv = 0.5*(q_cs.xy/q_cs.w) + 0.5;
 	vec2 vel = TexCoords - q_uv;
 */
-	// it should sample nearest (depth) in 3x3 neighbourhood
-	vec2 vel = textureLod(uVelocityBuf, TexCoords, 0.0).xy;
+	// TODO test later wether closestFragmentOffsetIn3x3() is worth it
```

Comment:
TOTALLY WORTH IT - see Karis TemporalAA pptx slide 27

However you can still dilate velocity if you have no velocity texture (which is what we should be striving towards).

Just when you run your 16x16 WG Compute Shader, compute Velocity for 18x18 tile (1 pixel border) in shared memory, then dilate (run your filter that's Z dependant) over the innermost 16x16 samples. Basically you'll be making a Tiled Renderer in software XD.